plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadata.java

```diff
@@ -454,6 +457,34 @@ public boolean supportsMissingColumnsOnInsert()
         return Optional.empty();
     }
 
+    @Override
+    public ColumnHandle getUpdateRowIdColumnHandle(ConnectorSession session, ConnectorTableHandle tableHandle, List<ColumnHandle> updatedColumns)
```

Comment:
Sorry, this method got renamed when support for SQL UPDATE was merged.  The new name of the method you were thinking of is `getDeleteRowIdColumnHandle(ConnectorSession session, ConnectorTableHandle tableHandle)`.  

However, for metadata delete, I don't believe either method will ever be called.

Reply:
Thanks, will try to remove this method then.